pkg/server/deploy/handlers.go

```diff
@@ -54,7 +55,7 @@ func (s *Service) Make(stream dpb.Deploy_MakeServer) error {
 	rs := bytes.NewReader(content.Bytes())
 	rc, err := s.ops.Deploy(u, appName, rs, description, s.options.RevisionHistoryLimit)
 	if err != nil {
-		return err
+		return teresa_errors.Get(err)
```

Comment:
I thought you could `return err` here because the log middleware does the unwrapping, am I right?